website/src/components/Navbar.astro

```diff
@@ -0,0 +1,17 @@
+
+<html  lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
```

Comment:
@Morteza-Mazrae I guess you need to remove the whole head here cause we already have it in the layout.